assets/scripts/im.pontoWikiaAppsBridge.js

```diff
@@ -27,8 +26,9 @@ define('im.pontoWikiaAppsBridge', ['ponto', 'im.window'], function (ponto, w) {
 			var target = event.target;
 
 			if (
-				target.tagName === 'A' &&
-				target.className.indexOf(poiArticleLinkClass) !== -1
+				target.tagName === 'A' || target.tagName === 'IMG' &&
```

Comment:
You probably want it to be `(target.tagName === 'A' || target.tagName === 'IMG') &&`